lib/spack/spack/fetch_strategy.py

```diff
@@ -940,6 +940,126 @@ def __str__(self):
         return '[git] {0}'.format(self._repo_info())
 
 
+@fetcher
+class CvsFetchStrategy(VCSFetchStrategy):
+
```

Comment:
Delete this empty line

Reply:
I think you deleted the wrong line, causing the style tests to fail.

Reply:
Corrected.